std_msgs/Header header

float32 roll	# roll increment {-1,0,1} [deg]
float32 pitch	# pitch increment {-1,0,1} [deg]
float32 zoom	# zoom increment {-1,0,1} [X]
uint8 action	# action {0=no action,1=take picture,2=start recording,3=stop recording} 
